{Release} Upgrade to Azure CLI 2.14.0 (#15637)
Azure/azure-cli: src/azure-cli/HISTORY.rst

Release History
===============

2.14.0
++++++

**AKS**

* Add PPG support (#15445)
* Update max standard load balancer timeout to 100 minutes (#15562)

**APIM**

* Fix issue with creating consumption tier instance (#15337)

**App Config**

* Fix querying key-values by comma separated labels (#15449)

**App Service**

* Bugfix: az webapp up fails when user doesn't have write permissions to project's parent directory (#15373)

> * Fix#-13777-Fix to remove escape chars from XML (#15364)

Reviewer: ```suggestion
* Fix #13777: Remove escape chars from XML (#15364)
```

> * Fixes #15441: az webapp create-remote-connection fails with AttributeError: 'Thread' object has no attribute 'isAlive' (#15446)

Reviewer 2: ```suggestion
* Fix #15441: az webapp create-remote-connection fails with AttributeError: 'Thread' object has no attribute 'isAlive' (#15446)
```

* [BREAKING CHANGE] az webapp up: add optional params (os & runtime) and updated runtimes (#15522)

**ARM**

* Make template deployment What-If commands GA (#15416)
* [BREAKING CHANGE] Add user confirmation for az ts create (#15480)

> * Fixed the returned data when tagging multiple resources (#15146)

Reviewer 2: ```suggestion
* Fix the returned data when tagging multiple resources (#15146)
```

**Backup**

> * `az backup policy create`: Added support for IaaSVM backup policy creation from CLI (#15542)

Reviewer 2: ```suggestion
* `az backup policy create`: Add support for IaaSVM backup policy creation from CLI (#15542)
```

* Increasing VM protection limit from 100 to 1000 (#15563)

**Compute**

* sig image-definition create: add --features (#15549)
* New API version of gallery_images 2020-09-30 (#15549)
* `az vm update / az sig image-version update`: Support update vm/image-version even it uses a cross tenant image (#15575)
* Remove validation of vm host SKUs (#15611)

**Cosmos DB**

* `az cosmosdb create/update`: Improve error message from incorrect --locations input (#15276)
* `az cosmosdb sql container create/update`: Add --analytical-storage-ttl parameter (#15276)

> **Docker**

Reviewer: Maybe this should go to **Misc.**?

* Add bash-completion to Dockerfile (#15228)

**HDInsight**

* [BREAKING CHANGE] az hdinsight create: remove two parameters: --public-network-access-type and  --outbound-public-network-access-type (#15582)

**IoT Central**

> * CLI: Removing preview warning since it is already GAed (#15527)

Reviewer 2: ```suggestion
* Remove preview warning since it is already GAed (#15527)
```

**Key Vault**

* Invalidate `--enable-soft-delete false` while creating or updating vaults (#15504)
* Make `--bypass` and `--default-action` work together with network acl parameters while creating vaults (#15571)

**RDBMS**

* Add List-SKUS Command, Table Transformers, Local Context for Postgres, MySQL, Mariadb Single Server (#15450)
* [BREAKING CHANGE] Parameter name updates. Improvements to Management Plane for MySQL and PostgreSQL (#15363)

> * : `az postgres|mariadb|mysql server create` : Updated create experience for Postgres, MySQL and MariaDB - new fields in the output , Introduce new values for `--public` parameter in create command (all,<IP>,<IPRange>,0.0.0.0) (#15538)

Reviewer 2: ```suggestion
* `az postgres|mariadb|mysql server create` : Update create experience for Postgres, MySQL and MariaDB - new fields in the output , Introduce new values for `--public` parameter in create command (all,<IP>,<IPRange>,0.0.0.0) (#15538)
```

* Add flexible-server connect command (#15419)

**SignalR**

* `az signalr create`: Add new option `--enable-messaging-logs` for controling service generate messaging logs or not (#15327)
* `az signalr update`: Add new option `--enable-messaging-logs` for controling service generate messaging logs or not (#15327)

**SQL**

* [BREAKING CHANGE] Fix response for backup storage redundancy param name and value for MI (#15367)
* `az sql db audit-policy show`: extend to show database's audit policy including LA and EH data (#15444)
* `az sql db audit-policy update`: extend to allow LA and EH update along with database's audit policy (#15444)
* `az sql db audit-policy wait`: place the CLI in a waiting state until a condition of the database's audit policy is met. (#15444)
* `az sql server audit-policy show`: extend to show servers's audit policy including LA and EH data (#15444)
* `az sql server audit-policy update`: extend to allow LA and EH update along with server's audit policy (#15444)
* `az sql server audit-policy wait`: place the CLI in a waiting state until a condition of the server's audit policy is met. (#15444)

> * Adding AAD-only Support for SQL Managed Instances and Servers (#15292)

Reviewer 2: ```suggestion
* Add AAD-only Support for SQL Managed Instances and Servers (#15292)
```
